src/go/rpk/pkg/cli/cmd/common/common.go

```diff
@@ -510,6 +558,39 @@ func AddTLSFlags(
 	return command
 }
 
+func AddAdminAPITLSFlags(
+	command *cobra.Command,
+	enableTLS *bool,
+	certFile, keyFile, truststoreFile *string,
+) *cobra.Command {
+	command.PersistentFlags().BoolVar(
+		enableTLS,
+		adminAPIEnableTLSFlag,
+		false,
+		"Enable TLS for the Admin API.",
```

Comment:
`Enable TLS for the Admin API (not necessary if specifying custom certs)`?

Reply:
Changed this one too.